reactioncommerce/reaction
# Consistent language in cart or orders

Resolves #2798 


## How to Test
1. Place an order
2. On the order completed page, under the "Your cart" section, you should now see "Items in order" instead of the former "Quantity Total" as shown in the image below:
<img width="348" alt="screen shot 2017-10-25 at 12 45 45" src="https://user-images.githubusercontent.com/12285426/31997092-a920ee7c-b982-11e7-8f97-25c4cb29ff6d.png">

3. Login as admin
4. Approve the order
5. On the Process Order Action View, you should now see "Items in order" instead of the former "Quantity Total" as shown below:
<img width="398" alt="screen shot 2017-10-25 at 12 46 14" src="https://user-images.githubusercontent.com/12285426/31997098-ae4a3318-b982-11e7-8b57-528094bd8658.png">
